rfcs/supported/algorithms_histogram_cpu_backends/README.md

```diff
@@ -1,18 +1,131 @@
 # Host Backends Support for the Histogram APIs
 
 ## Introduction
-The oneDPL library added histogram APIs, currently implemented only for device policies with the DPC++ backend. These
-APIs are defined in the oneAPI Specification 1.4. Please see the
+The oneDPL library added histogram APIs in the oneAPI Specification 1.4. Please see the
 [oneAPI Specification](https://github.com/uxlfoundation/oneAPI-spec/blob/main/source/elements/oneDPL/source/parallel_api/algorithms.rst#parallel-algorithms)
-for details. The host-side backends (serial, TBB, OpenMP) are not yet supported. This RFC proposes extending histogram
-support to these backends.
-
-The pull request for the proposed implementation exists [here](https://github.com/oneapi-src/oneDPL/pull/1974).
+for details. This RFC outlines the now supported host-side backends (serial, TBB, OpenMP).
+
+The commit containing the implementation proposed here can be found
+[here](https://github.com/uxlfoundation/oneDPL/commit/780705c3bf3eee50b72a37cbc730de28d524efe9).
+
+### Version Information
+Histogram APIs are supported for host execution policies as of version 2022.8.0 of oneDPL, specification version 1.4.
+```
+ONEDPL_VERSION_MAJOR=2022
+ONEDPL_VERSION_MINOR=8
+ONEDPL_VERSION_PATCH=0
+
+ONEDPL_SPEC_VERSION=104
+```
+
+Histogram APIs are supported for device execution policies as of version 2022.5.0.  It was first specified in oneDPL
+specification version 1.4. However, this was released before `ONEDPL_SPEC_VERSION` was defined within our implementation
+of oneDPL.
+```
+ONEDPL_VERSION_MAJOR=2022
+ONEDPL_VERSION_MINOR=5
+ONEDPL_VERSION_PATCH=0
+```
 
 ## Motivations
 There are many cases to use a host-side serial or a host-side implementation of histogram. Another motivation for adding
 the support is simply to be spec compliant with the oneAPI specification.
```

Comment:
This does not seem really informative, I think it can be deleted.

Reply:
removed